.github/workflows/R-CMD-check-bioc.yaml

```
@@ -0,0 +1,121 @@
+# Based on workflows from https://github.com/Huber-group-EMBL/rhdf5
+# Uses steps from https://github.com/grimbough/bioc-actions
+on:
+  push:
+    branches:
+      - devel
+  pull_request:
+    branches:
+      - devel
```

do we not want to run the ci when merging one branch into another? 

```suggestion
  pull_request:
```